av/video/codeccontext.pyx

```diff
@@ -159,3 +159,10 @@ cdef class VideoCodecContext(CodecContext):
     property coded_height:
         def __get__(self):
             return self.ptr.coded_height
+
+    property color_range:
```

Comment:
I know we haven't done this everywhere, but let's use the `@property` decorator going forward

Reply:
Perhaps these should be handled the same as on the Frame where they are enums.

Reply:
I've changed this to an `@property` instead and added a brief docstring. Should the docstring be more elaborate?